search-commons/src/main/java/no/unit/nva/search/models/SearchResourcesResponse.java

```diff
@@ -148,6 +151,27 @@ public SearchResourcesResponse build() {
 
     }
 
+    public static SearchResourcesResponse fromSearchResponse(SearchResponse searchResponse, URI id) {
+        List<JsonNode> sourcesList = extractSourcesList(searchResponse);
+        Long total = searchResponse.getHits().getTotalHits().value;
```

Comment:
use primitives

Reply:
Because SearchResponse has long and our SearchResourcesResponse has int. I wanted to change our class to use long as well.

Reply:
Done